regress/unittests/win32compat/tests.h

```diff
@@ -8,6 +8,6 @@ void delete_dir_recursive(char *full_dir_path);
 
 #define ASSERT_HANDLE(handle,retValue) \
 { \
-	ASSERT_PTR_NE(handle, INVALID_HANDLE_VALUE); \
-	ASSERT_PTR_NE(handle, 0); \
+	retValue = ((handle != INVALID_HANDLE_VALUE) && (handle != NULL)) ? 0 : -1; \
+	ASSERT_INT_EQ(retValue, 0); \
```

Comment:
reverting this.?
If ASSERT_INT_EQ(retValue, 0) fails, you don't know upfront (from test execution output) whether its due to handle is invalid_handle or NULL. 
But with 2 explicit asserts, you know if handle was null or invalid_handle.